### ISSUE TITLE
Add AZ::ScheduledSystemEvent or allow AZ::ScheduledEvent to fire outside of TickBus

### ISSUE THREAD

**Issue by AMZN-Olex** [o3de/o3de#7988]
**Is your feature request related to a problem? Please describe.**
When the Editor is not in focus, no TickBus events occur. Thus, no AZ::ScheduledEvent occur either. A lot of networking/multiplayer logic depends on the functionality of AZ::ScheduledEvent, including sending packets or sending level data to the Editor server. 

One of the issue this causes is usability of running multiplayer game in the Editor, where the Editor would launch the server process in the foreground. The server would launch but remain empty because the Editor is frozen and not ticking, thus the Multiplayer gem wouldn't send the level data to the server until you alt-tab back to the Editor.

**Describe the solution you'd like**
Either add AZ::ScheduledSystemEvent to support ticking outside of TickBus when the Editor is frozen, or have a configurable option on AZ::ScheduledEvent, or perhaps the Editor should tick TickBus and AZ::ScheduledEvent's even when it's not in focus?

**Describe alternatives you've considered**
Perhaps, change AZ::ScheduledEvent to tick from SystemTickBus instead or some other PAL option and avoid basing on any EBus entirely?

**Comment by lmbr-pip** [comment]
Looking for input from sig-core, who own this code.

**Comment by junbo75** [comment]
Just got a repro of the same issue in the stabilization branch with @AMZN-Gene. We may need to escalate the priority of this since the workaround for it is not obvious and it could mislead developers to debug into the networking connection when testing the editor server.

**Comment by AMZN-Gene** [comment]
Agreed, this is very visible. Every new users will learn the hard way that they need to click back in the editor after the server is launched.
